int32 tryNumber
---
string recognizedName
